diff -Nur pidgin-2.3.1/doc/pidgin.1.in pidgin-2.3.1.new/doc/pidgin.1.in
--- pidgin-2.3.1/doc/pidgin.1.in	2007-12-07 09:36:59.000000000 -0500
+++ pidgin-2.3.1.new/doc/pidgin.1.in	2008-01-29 18:53:30.000000000 -0500
@@ -19,9 +19,9 @@
 .\" License along with this manual; if not, write to the Free
 .\" Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
 .\" Boston, MA  02111-1301  USA.
-.TH pidgin 1
+.TH pidgin 1 "" "Pidgin v@VERSION@"
 .SH NAME
-Pidgin v@VERSION@ \- Instant Messaging client
+pidgin \- Pidgin Instant Messaging client
 .SH SYNOPSIS
 .TP 5
 \fBpidgin \fI[options]\fR
