--- pidgin-2.0.0+beta5/configure.orig	2006-12-11 13:11:56.000000000 -0500
+++ pidgin-2.0.0+beta5/configure	2006-12-11 12:17:51.000000000 -0500
@@ -32122,7 +32122,7 @@
 	enable_nss="no"
 
 	if test "x$nss_manual_check" = "xno"; then
-		if `$PKG_CONFIG --exists mozilla-nss`; then
+		if `$PKG_CONFIG --exists xulrunner-nss`; then
 
 pkg_failed=no
 echo "$as_me:$LINENO: checking for NSS" >&5
@@ -32133,12 +32133,12 @@
         pkg_cv_NSS_CFLAGS="$NSS_CFLAGS"
     else
         if test -n "$PKG_CONFIG" && \
-    { (echo "$as_me:$LINENO: \$PKG_CONFIG --exists --print-errors \"mozilla-nss\"") >&5
-  ($PKG_CONFIG --exists --print-errors "mozilla-nss") 2>&5
+    { (echo "$as_me:$LINENO: \$PKG_CONFIG --exists --print-errors \"xulrunner-nss\"") >&5
+  ($PKG_CONFIG --exists --print-errors "xulrunner-nss") 2>&5
   ac_status=$?
   echo "$as_me:$LINENO: \$? = $ac_status" >&5
   (exit $ac_status); }; then
-  pkg_cv_NSS_CFLAGS=`$PKG_CONFIG --cflags "mozilla-nss" 2>/dev/null`
+  pkg_cv_NSS_CFLAGS=`$PKG_CONFIG --cflags "xulrunner-nss" 2>/dev/null`
 else
   pkg_failed=yes
 fi
@@ -32151,12 +32151,12 @@
         pkg_cv_NSS_LIBS="$NSS_LIBS"
     else
         if test -n "$PKG_CONFIG" && \
-    { (echo "$as_me:$LINENO: \$PKG_CONFIG --exists --print-errors \"mozilla-nss\"") >&5
-  ($PKG_CONFIG --exists --print-errors "mozilla-nss") 2>&5
+    { (echo "$as_me:$LINENO: \$PKG_CONFIG --exists --print-errors \"xulrunner-nss\"") >&5
+  ($PKG_CONFIG --exists --print-errors "xulrunner-nss") 2>&5
   ac_status=$?
   echo "$as_me:$LINENO: \$? = $ac_status" >&5
   (exit $ac_status); }; then
-  pkg_cv_NSS_LIBS=`$PKG_CONFIG --libs "mozilla-nss" 2>/dev/null`
+  pkg_cv_NSS_LIBS=`$PKG_CONFIG --libs "xulrunner-nss" 2>/dev/null`
 else
   pkg_failed=yes
 fi
@@ -32175,9 +32175,9 @@
         _pkg_short_errors_supported=no
 fi
         if test $_pkg_short_errors_supported = yes; then
-	        NSS_PKG_ERRORS=`$PKG_CONFIG --short-errors --errors-to-stdout --print-errors "mozilla-nss"`
+	        NSS_PKG_ERRORS=`$PKG_CONFIG --short-errors --errors-to-stdout --print-errors "xulrunner-nss"`
         else
-	        NSS_PKG_ERRORS=`$PKG_CONFIG --errors-to-stdout --print-errors "mozilla-nss"`
+	        NSS_PKG_ERRORS=`$PKG_CONFIG --errors-to-stdout --print-errors "xulrunner-nss"`
         fi
 	# Put the nasty error message in config.log where it belongs
 	echo "$NSS_PKG_ERRORS" >&5
@@ -32202,8 +32202,8 @@
 				have_nss="yes"
 
 fi
-			mozilla_nspr="mozilla-nspr"
-			mozilla_nss="mozilla-nss"
+			mozilla_nspr="xulrunner-nspr"
+			mozilla_nss="xulrunner-nss"
 		else
 			if `$PKG_CONFIG --exists nss`; then
 
