--- pidgin.orig/libpurple/protocols/yahoo/yahoo_picture.c	2006-12-12 20:59:01.000000000 -0500
+++ pidgin/libpurple/protocols/yahoo/yahoo_picture.c	2006-12-12 21:00:28.000000000 -0500
@@ -130,9 +130,6 @@
 		if (url_data != NULL) {
 			yd = gc->proto_data;
 			yd->url_datas = g_slist_prepend(yd->url_datas, url_data);
-		} else {
-			g_free(data->who);
-			g_free(data);
 		}
 	} else if (who && send_icon_info) {
 		yahoo_send_picture_info(gc, who);
--- pidgin.orig/libpurple/protocols/yahoo/yahoo_profile.c	2006-12-12 20:59:01.000000000 -0500
+++ pidgin/libpurple/protocols/yahoo/yahoo_profile.c	2006-12-12 21:02:13.000000000 -0500
@@ -928,11 +928,6 @@
 				FALSE, yahoo_got_photo, info2_data);
 		if (url_data != NULL)
 			yd->url_datas = g_slist_prepend(yd->url_datas, url_data);
-		else {
-			g_free(info2_data->info_data->name);
-			g_free(info2_data->info_data);
-			g_free(info2_data);
-		}
 	} else {
 		/* Emulate a callback */
 		yahoo_got_photo(NULL, info2_data, NULL, 0, NULL);
@@ -1272,10 +1267,6 @@
 	url_data = gaim_util_fetch_url(url, TRUE, NULL, FALSE, yahoo_got_info, data);
 	if (url_data != NULL)
 		yd->url_datas = g_slist_prepend(yd->url_datas, url_data);
-	else {
-		g_free(data->name);
-		g_free(data);
-	}
 
 	g_free(url);
 }
