diff -Nur pidgin-2.0.0~beta6+mtn20070409/libpurple/purple-remote pidgin-2.0.0~beta6+mtn20070409.new/libpurple/purple-remote
--- pidgin-2.0.0~beta6+mtn20070409/libpurple/purple-remote	2007-04-09 22:20:12.000000000 -0400
+++ pidgin-2.0.0~beta6+mtn20070409.new/libpurple/purple-remote	2007-04-09 22:21:12.000000000 -0400
@@ -1,6 +1,10 @@
 #!/usr/bin/python
 
-import dbus
+try:
+    import dbus
+except:
+    print "DBus library not installed. Please install the python-dbus package."
+    raise
 import re
 import urllib
 import sys
@@ -61,6 +65,7 @@
     GaimAccountsFindConnected?name=&protocol=prpl-jabber
     GaimAccountFindConnected(,prpl-jabber)
 """ % sys.argv[0]
+    sys.exit(1)
 
 cgaim = CheckedObject(gaim)
 
@@ -204,7 +209,7 @@
             show_help()
             raise "Unknown command: %s" % command
 
-if len(sys.argv) == 1:
+if len(sys.argv) == 1 or sys.argv[1] == "--help" or sys.argv == "-h":
     show_help()
 
 for arg in sys.argv[1:]:
diff -Nur pidgin-2.0.0~beta6+mtn20070409/libpurple/purple-url-handler pidgin-2.0.0~beta6+mtn20070409.new/libpurple/purple-url-handler
--- pidgin-2.0.0~beta6+mtn20070409/libpurple/purple-url-handler	2007-04-09 22:16:28.000000000 -0400
+++ pidgin-2.0.0~beta6+mtn20070409.new/libpurple/purple-url-handler	2007-04-09 22:20:36.000000000 -0400
@@ -1,6 +1,10 @@
 #!/usr/bin/python
 
-import dbus
+try:
+    import dbus
+except:
+    print "DBus library not installed. Please install the python-dbus package."
+    raise
 import re
 import sys
 import time
@@ -254,7 +258,7 @@
 
 
 def main(argv=sys.argv):
-    if len(argv) != 2:
+    if len(argv) != 2 or argv[1] == "--help" or argv[1] == "-h":
         print "Usage: %s URI" % argv[0]
         print "Example: %s \"xmpp:romeo@montague.net?message\"" % argv[0]
         return
