diff -Nur pidgin-2.3.1/Makefile.in pidgin-2.3.1.new/Makefile.in
--- pidgin-2.3.1/Makefile.in	2007-12-10 09:50:05.000000000 -0500
+++ pidgin-2.3.1.new/Makefile.in	2007-12-15 19:22:16.000000000 -0500
@@ -875,7 +875,7 @@
 @HAVE_DOXYGEN_TRUE@	@echo "Running doxygen..."
 @HAVE_DOXYGEN_TRUE@	@doxygen
 @HAVE_DOXYGEN_TRUE@@HAVE_XSLTPROC_TRUE@	@echo "Generating devhelp index..."
-@HAVE_DOXYGEN_TRUE@@HAVE_XSLTPROC_TRUE@	@xsltproc doxy2devhelp.xsl doc/xml/index.xml > doc/html/pidgin.devhelp
+@HAVE_DOXYGEN_TRUE@@HAVE_XSLTPROC_TRUE@	@xsltproc $(top_srcdir)/doxy2devhelp.xsl doc/xml/index.xml > doc/html/pidgin.devhelp
 @HAVE_DOXYGEN_TRUE@@HAVE_XSLTPROC_TRUE@	@echo "(Symlink doc/html to ~/.local/share/gtk-doc/html/pidgin to make devhelp see the documentation)"
 @HAVE_DOXYGEN_TRUE@@HAVE_XSLTPROC_FALSE@	@echo "Not generating devhelp index: xsltproc was not found by configure"
 @HAVE_DOXYGEN_FALSE@	@echo "doxygen was not found during configure.  Aborting."
