Description: Fix path to libnssckbi.so
Author: Mike Hommey <mh+reportbug@glandium.org>
Bug-Debian: http://bugs.debian.org/564711
Forwarded: no
Index: pidgin-2.7.1/libpurple/plugins/ssl/ssl-nss.c
===================================================================
--- pidgin-2.7.1.orig/libpurple/plugins/ssl/ssl-nss.c
+++ pidgin-2.7.1/libpurple/plugins/ssl/ssl-nss.c
@@ -131,7 +131,7 @@ ssl_nss_init_nss(void)
 
 	/* TODO: Fix this so autoconf does the work trying to find this lib. */
 #ifndef _WIN32
-	lib = g_strdup(LIBDIR "/libnssckbi.so");
+	lib = g_strdup("libnssckbi.so");
 #else
 	lib = g_strdup("nssckbi.dll");
 #endif
